examples/shell/shell_common/BUILD.gn

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+import("//build_overrides/openthread.gni")
+import("${chip_root}/src/platform/device.gni")
+
+config("chip_shell_collection_config") {
+  include_dirs = [
+    ".",
+    "./include",
+    "${chip_root}/src/lib/shell",
+  ]
+
+  defines = [
+    "ENABLE_CHIP_SHELL",
+    "OPENTHREAD_CONFIG_CLI_TRANSPORT=OT_CLI_TRANSPORT_CONSOLE",
+  ]
+}
+
+static_library("chip_shell_collection") {
```

Comment:
Why not "shell_common" ?

Generally it's nice to name libraries that have dedicated directories the same as the directory, since that makes GN's shorthand work (you can just depend on the directory).

Reply:
Done